charts/pega/README.md

```diff
@@ -950,12 +950,35 @@ Parameter   | Description   | Default value
 - You must use the latest Docker web images in order to use this feature; if you use Helm chart version `2.2.0`, with outdated Docker images and set `service.tls.enabled` to `true`, the deployment logs a `Bad Gateway` error. Helm chart version `2.2.0`, you must update your Pega Platform version to the latest patch version or set `service.tls.enabled` to `false`.
 
 #### Example:
+With TLS enabled for the web tier and the traefik addon deployed for `k8s` provider and the keystore file and password are from external secret operator, you set the following parameters in the `values.yaml`:
```

Comment:
slight tweak>>
With TLS enabled for the web tier and the traefik addon deployed for `k8s` provider and the keystore file and password are from external secret operator, you set the following parameters in the `values.yaml`:
to 
After you enable TLS for the web tier, deploy the traefik addon for `k8s` provider, and configure the keystore file and password using the external secret operator, set the following parameters in the `values.yaml`:

Reply:
done